testing/test_aux.out

```diff
@@ -57,6 +57,9 @@ RESULT pass security.passwords.telnet Default passwords have been changed.
 RESULT pass security.passwords.ssh Default passwords have been changed.
 RESULT skip security.firmware Could not retrieve a firmware version with nmap. Check bacnet port.
 RESULT pass security.firmware version found: ?\xFF\xFF\x19,>u\x08\x00no
+RESULT pass security.ssh.version Device only supports SSHv2
```

Comment:
There should be { pass, skip, fail } coverage.
You should consider putting these in test_modules.sh -- it's a bit quicker to setup and develop than test_aux.sh

Reply:
Thanks @grafnu, I've added coverage for pass, skip and fail. I've also removed the tests from test_aux and added them in to test_modules